Auth0/CredentialsManager.swift

```diff
@@ -41,9 +41,11 @@ public struct CredentialsManager {
     /// - Parameters:
     ///   - authentication: Auth0 authentication instance
     ///   - storeKey: Key used to store user credentials in the keychain, defaults to "credentials"
-    public init(authentication: Authentication, storeKey: String = "credentials") {
+    ///   - storage: The A0SimpleKeychain object that user credentials are saved to. Defaults to the standard application keychain
```

Comment:
///   - storage: The A0SimpleKeychain instance used to manage credentials storage. Defaults to a standard A0SimpleKeychain instance